SlicerLite/Resources/Home.qrc

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE RCC>
+<!--
+  This file was automatically generated by genqrc.py ['./Icons/']
+  See http://www.slicer.org/slicerWiki/index.php/Documentation/Nightly/Developers/Build_system/Qt_resource_files for more information.
+-->
+
+<RCC version="1.0">
+
+  <qresource prefix="Icons">
+    <file alias="delete.png">./Icons/delete.png</file>
```

Comment:
QRC file seems not included in CMakeLists and content seems wrong.
Probably to remove if unused.